<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Choose_File_Label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Нажмите кнопку обзор, чтобы выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="Actions_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Действия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Input_Params">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Predict_Label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите значения следующим образом: a; b; c; ... в соответсвии с количеством признаков</string>
    </property>
   </widget>
   <widget class="QLabel" name="Predict_Label2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>901</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="File_Name_Out">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Select_File_Button">
       <property name="text">
        <string>Обзор...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calculate_Button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Plot_Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать граффик</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Сбросить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Predict_Button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string> Предсказать</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Plot_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>691</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MSE_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="MAE_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
